### ISSUE TITLE
AMI used in templates/bastion.json does not exist

### ISSUE THREAD

**Issue by pmather** [hybox/aws#88]
When launching the Hydra-in-a-Box stack in the us-west-2 region, deployment failed in the `bastion` part of the CloudFormation stack.  I received this error:

```
AMI ami-004b0f60 is invalid: The image id '[ami-004b0f60]' does not exist
```

The AMIs are mapped in `templates/bastion.json`:

```
  "Mappings" : {
    "AWSRegionToAMI" : {
      "us-east-1"      : { "AMIID" : "ami-4868ab25" },
      "us-west-1"      : { "AMIID" : "ami-a275b1c2" },
      "us-west-2"      : { "AMIID" : "ami-004b0f60" },
      "eu-west-1"      : { "AMIID" : "ami-a8dd45db" },
      "ap-northeast-1" : { "AMIID" : "ami-2443b745" },
      "ap-southeast-2" : { "AMIID" : "ami-53371f30" },
      "ap-southeast-1" : { "AMIID" : "ami-a79b49c4" },
      "eu-central-1"   : { "AMIID" : "ami-5825cd37" }
    }
  },
```

AMI ami-004b0f60 does not currently exist in the us-west-2 region.

**Comment by cbeer** [comment]
It looks like AWS has since made a new set of AMIs available: https://aws.amazon.com/amazon-linux-ami/

**Comment by yinlinchen** [comment]
Should be 
"us-west-1"      : { "AMIID" : "ami-004b0f60" },
"us-west-2"      : { "AMIID" : "ami-a275b1c2" },

**Comment by thatbudakguy** [comment]
AMI ids as of 9/27/16 as per @cbeer link above, working for me:
```
"us-west-1": { "AMIID": "ami-23e8a343" },
"us-west-2": { "AMIID": "ami-5ec1673e" },
```

**Comment by yinlinchen** [comment]
@thatbudakguy are you saying it is still working or it was working on 9/27/16?

**Comment by thatbudakguy** [comment]
@yinlinchen still working as far as I can tell, but I haven't completed a deployment due to #95 - just haven't seen any `ami does not exist` errors.

**Comment by yinlinchen** [comment]
@thatbudakguy I have been confused by your "9/27/16". You are using the current set of AMIs. The issue here is that bastion.json needs to be updated, and these AMIs https://github.com/hybox/aws/blob/master/templates/bastion.json#L37 are no longer exist.

**Comment by eefahy** [comment]
Closed by c48130f621c14befaafd89f890a40ea1e1a2e39b

**Comment by mjgiarlo** [comment]
Thanks, @pmather @yinlinchen @thatbudakguy!
